projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -28,7 +28,7 @@ import { CommentSetComponent } from '../../annotations/comment-set/comment-set.c
 @Component({
   selector: 'mv-pdf-viewer',
   templateUrl: './pdf-viewer.component.html',
-  styleUrls: ['./pdf-viewer.component.scss'],
+  styleUrls: ['./pdf-viewer.component.scss', '../../media-viewer.component.scss'],
```

Comment:
why do u need this two css imports? try using incapsulate.none to inherit the css from media viewer component